pkgs/top-level/python-packages.nix

```diff
@@ -3873,7 +3873,9 @@ in {
 
   monotonic = callPackage ../development/python-modules/monotonic { };
 
-  MySQL_python = callPackage ../development/python-modules/mysql_python { };
+  MySQL_python = callPackage ../development/python-modules/mysql_python {
```

Comment:
I opened a pr to remove this package: https://github.com/NixOS/nixpkgs/pull/68354